Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1051,31 +1051,22 @@ def genS(fragment,howMuch):
 U2000by1={'--relval': '2000,1'}
 U80by1={'--relval': '80,1'}
 
-hiAlca2011 = {'--conditions':'auto:run1_mc_hi'}
-hiAlca2015 = {'--conditions':'auto:run2_mc_hi', '--era':'Run2_HI'}
-hiAlca2017 = {'--conditions':'auto:phase1_2017_realistic', '--era':'Run2_2017_pp_on_XeXe'}
-hiAlca2018 = {'--conditions':'auto:phase1_2018_realistic_hi', '--era':'Run2_2018'}
-hiAlca2018_ppReco = {'--conditions':'auto:phase1_2018_realistic_hi', '--era':'Run2_2018_pp_on_AA'}
-hiAlca2021_ppReco = {'--conditions':'auto:phase1_2022_realistic_hi', '--era':'Run3_pp_on_PbPb'}
-hiAlca2021_ppReco_approxClusters = {'--conditions':'auto:phase1_2022_realistic_hi', '--era':'Run3_pp_on_PbPb_approxSiStripClusters'}
-
-
-hiDefaults2011=merge([hiAlca2011,{'--scenario':'HeavyIons','-n':2}])
-hiDefaults2015=merge([hiAlca2015,{'--scenario':'HeavyIons','-n':2}])
-hiDefaults2017=merge([hiAlca2017,{'-n':2}])
-hiDefaults2018=merge([hiAlca2018,{'--scenario':'HeavyIons','-n':2}])
-hiDefaults2018_ppReco=merge([hiAlca2018_ppReco,{'-n':2}])
-hiDefaults2021_ppReco=merge([hiAlca2021_ppReco,{'-n':2}])
-hiDefaults2021_ppReco_approxClusters=merge([hiAlca2021_ppReco_approxClusters,{'-n':2}])
+hiDefaults2011 = {'--conditions':'auto:run1_mc_hi','--scenario':'HeavyIons'}
```

Comment:
Is this update needed? The PR description seems to be only for Run-3. It does not cover `scenario` updates on old workflow.

Reply:
Hi @srimanob  
I decided take the opportunity to clean up the code a little bit to remove this intermediate `hiAlca` thing.
I have just double-checked the output driver commands from the 2011 heavy-ion workflows.  They contain `--scenario HeavyIons` both before and after this pull request. 
The comparisons only seem to show a non-trivial difference in 159.3, which is a workflow where this PR changes the GT.

Reply:
@srimanob are you ok with my response here?

Reply:
Oh, yeah. I am fine with it, thx.